src/components/forms/formComponents.js

```diff
@@ -276,7 +273,7 @@ export const FormInputSelect = (config) => {
   const optionsAreString = config.optionsAreString || (!isNil(selectOptions) && isString(selectOptions[0]));
   const normalizedOptions = (!isNil(selectOptions) ? normalizeSelectOptions(selectOptions, optionsAreString) : undefined);
 
-  return h(component, {
+  return React.createElement(component, {
```

Comment:
I think this can also be simplified using the fragment operator: `<>` `</>`

Reply:
This was challenging - I ended up letting co-pilot do the conversion (`convert to jsx fragment`) for me.